charts/super-agent/charts/super-agent-deployment/templates/_helpers.tpl

```diff
@@ -2,19 +2,80 @@
 Return the name of the configMap holding the Super Agent's config. Defaults to release's fill name suffiexed with "-config"
 */ -}}
 {{- define "newrelic-super-agent.config.name" -}}
-{{- (include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" "local-data" "suffix" "superagent-config" )) -}}
+{{- include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" "local-data" "suffix" "superagent-config" ) -}}
 {{- end -}}
 
+
+
+{{- /*
+Return the agents part that should go in the super agent config. It is created from `.Values.config.subAgents`.
```

Comment:
I think that the comment is wrong. It creates the list of subAgents that is used for both the SA config and the SubAgents ones

Reply:
You are right. And not only that is that it was creating the agents incorrectly.

Fixed on [#7bb25f2](https://github.com/newrelic/helm-charts/pull/1466/commits/7bb25f25b56aa0f16e96d8b4c94d9f75a0f405ac)

Let me know what you think.